lib/ufe/StagesSubject.cpp

```diff
@@ -247,13 +252,19 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 				Ufe::Attributes::notify(ufePath, changedPath.GetName());
 			}
 		}
+
+		// Send a special message when visibility has changed.
+		if (changedPath.GetNameToken() == UsdGeomTokens->visibility)
+		{
+			Ufe::VisibilityChanged vis(ufePath);
+			Ufe::Object3d::notify(vis);
+		}
```

Comment:
Send new Ufe::VisibilityChanged notif when the USD attribute 'visibility' changes.